test/test_admin.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase, Client
+from doogkingapp.models import Profile
+
+
+class SuperUserTestCase(TestCase):
+
+    def create_superuser(self):
+        return Profile.objects.create_superuser(email="superusertest@gmail.com", password="password123", is_staff=True,
```

Comment:
S'han de canviar totes les contrasenyes de test afegint-hi una majúscula, que ara hi ha la comprovació a "managers.py" i si no tots els tests que creen una conta peten XD
```suggestion
        return Profile.objects.create_superuser(email="superusertest@gmail.com", password="Password123", is_staff=True,
```

Reply:
Els tests aquí no estan actualitzats. A la branca de test estan els últims amb totes les modificacions.